dispatches/case_studies/renewables_case/run_double_loop.py

```diff
@@ -32,7 +32,7 @@
 from pyomo.common.fileutils import this_file_dir
 import pandas as pd
 from pathlib import Path
-from dispatches_sample_data import rts_gmlc
+from dispatches_data.api import path
```

Comment:
It is related in the sense that it's a function that returns a `pathlib.Path` object: https://dispatches-data-packages.readthedocs.io/en/latest/#dispatches_data.api.path.

As for whether the name is too similar: I think in general it's a fair point. I think I chose `path()` as opposed to e.g. `get_path()` to more closely resemble the [`importlib.resources`](https://docs.python.org/3/library/importlib.resources.html) API, where the "getters" are also named without the `get_` prefix.

I'm open to revisit this (or add an alias, to preserve compatibility?), but this would have to be done in the [gmlc-dispatches/data-packages](https://github.com/gmlc-dispatches/data-packages) repository first.

Reply:
I'm okay with it